src/vivliostyle/selectors.js

```diff
@@ -181,4 +183,231 @@ goog.scope(function() {
      */
     vivliostyle.selectors.fragmentIndices = {};
 
+
+    /**
+     * @param {Element} sourceNode
+     * @param {adapt.vgen.PseudoelementStyler} styler
+     * @constructor
+     */
+    vivliostyle.selectors.AfterIfContinues = function(sourceNode, styler) {
+        /** @const */ this.styler = styler;
+        /** @const */ this.sourceNode = sourceNode;
+    };
+    /** @const */ var AfterIfContinues = vivliostyle.selectors.AfterIfContinues;
+
+
+    /**
+     * @param {!adapt.layout.Column} column
+     * @param {!adapt.vtree.NodeContext} parentNodeContext
+     * @return {!adapt.task.Result.<!Element>}
+     */
+    AfterIfContinues.prototype.createElement = function(column, parentNodeContext) {
+        var doc = parentNodeContext.viewNode.ownerDocument;
+        var fragment = doc.createDocumentFragment();
```

Comment:
Fix it in [this commit](https://github.com/vivliostyle/vivliostyle.js/pull/348/commits/59c5adcc7392ee273b8220fa13e970e90c756b4e).